microhapulator/filter.py

```diff
@@ -125,6 +129,11 @@ def keep(self, read):
         is_ambiguous = read.seq.count("N") / len(read.seq) > self.threshold
         return not is_ambiguous
 
-    @property
-    def summary(self):
-        return f"ReadsRemoved\tReadsKept\n{self.num_reads_failed}\t{self.num_reads_passed}"
+
+class LengthSingleReadFilter(SingleReadFilter):
+    def __init__(self, reads_in, outfile, threshold=50):
+        self.threshold = threshold
+        super().__init__(reads_in, outfile)
+
+    def keep(self, read):
+        return len(read) > self.threshold
```

Comment:
The conciseness of this solution is a testament to the refactoring work we put in to #165!